cypress/e2e/style-test.cy.js

```diff
@@ -0,0 +1,42 @@
+const {parsePxInt} = require('../../modules/pixels');
+
+const testsLocation = process.env.TESTS_LOCATION || 'style.tests.json'
+const STYLETESTS = require('../../'+ testsLocation);
+STYLETESTS.tests.forEach(test => {
+	describe(test.name, () => {
+		it(test.name + ' passes', () => {
+			if (test.viewport) {
+				// https://docs.cypress.io/api/commands/viewport
+				const viewportIsString = typeof test.viewport === 'string';
+				if (viewportIsString) {
+					cy.viewport(test.viewport);
+				} else if (Array.isArray(test.viewport) && test.viewport.length === 2) {
+					cy.viewport(test.viewport[0], test.viewport[1]);
+				}
+			}
+			cy.visit(test.url)
+			test.styles.forEach(styleSet => {
```

Comment:
[note] be aware that `forEach` is not intrinsically async — I don't think it matters in this case, but if you see weird behavior, it might be worth knowing 👍